src/pages/index.js

```diff
@@ -16,11 +16,11 @@ const features = [
       "New to Camunda 8? Create an account and start modeling your first process.",
   },
   {
-    title: "Components",
+    title: "Using Camunda",
     imageUrl: "img/home-components.png",
     url: "/docs/components/",
```

Comment:
Will this URL change or need to change as well?

Reply:
Eventually. We should probably discuss timing since it will create a lot of URL and directory changes (and redirects...).

Reply:
I was going to say…my renaming PTSD spidey senses are tingling 🙃